public/components/application_analytics/components/flyout_components/trace_detail_render.tsx

```diff
@@ -36,6 +37,17 @@ export const TraceDetailRender = ({
   const [payloadData, setPayloadData] = useState('');
   const [colorMap, setColorMap] = useState({});
 
+  const storedFilters = sessionStorage.getItem('TraceAnalyticsSpanFilters');
+  const [spanFilters, setSpanFilters] = useState<TraceFilter[]>(
+    storedFilters ? JSON.parse(storedFilters) : []
```

Comment:
nit to avoid parsing again on re-render
```suggestion
    () => storedFilters ? JSON.parse(storedFilters) : []
```